README.md

```diff
@@ -80,15 +79,70 @@ Depending on the scenario a DomainError will be returned stating the problem. If
 A payment will time out within 35 minutes, meaning that the whole process of user accepting, callbacks made and authorization must be completed within 35 minutes.
 Furthermore after you get the callback containing the card data, you must update the status of the authorization to either "authorize-succesfull" or "authorize-failed" within 32 seconds to ensure a smooth experience for the user waiting for the confirmation.
 
-## Dankort acquiring
+## Strong Customer Authentication (SCA)
+
+We aim to ensure Delegated Authentication (DA). This means that responsability for authenticating the customer/payer no longer lies with the Issuer, but is delegated to MobilePay.  When/if we fail, and the Issuer is responding to an authrorisation attempt with a Soft Decline/"step-up", a 3DSecure fallback solution must be in place.
```

Comment:
No guidance on 3DS version, right? 🙂

Just a few typos:
```suggestion
We aim to ensure Delegated Authentication (DA). This means that responsibility for authenticating the customer/payer no longer lies with the Issuer, but is delegated to MobilePay.  When/if we fail, and the Issuer is responding to an authorisation attempt with a Soft Decline/"step-up", a 3-D Secure fallback solution must be in place.
```

Reply:
Thanks for fixing the typos. No guidelines from us regarding version of 3DS, any will do in this solution.

Reply:
Permaybehaps it is worth mentioning that if you (the PSP) defaults to a 3DSv1 flow and the issuer declines the authorization with "step-up" it's probably worth either retrying 3DSv1 (if you hope the issuer keeps state and runs an SCA compliant 3DSv1 flow (rumors says that issuers will allow to do so) the second time) or 3DSv2 (if the issuer supports 3DSv2). Though, there are incentives (read: fees, rules, approval rates, etc.) depending on the acquirer, scheme, 3DS version, etc. so ... 🤷‍♂️

However, at least it will be worth mentioning if you have bilateral agreements of some sort wrt. SCA 😇